<p>Compare strings and bignums with the equality-operators, instead of the _is/_isnt operators.</p>
<h2>Noncompliant Code Example</h2>
<pre>
a _is "test"
</pre>
<h2>Compliant Solution</h2>
<pre>
a = "test"
</pre>
